data/categories.txt

```diff
@@ -2126,7 +2126,7 @@ fr:Arrêt de bus|2bus|arrêt|2autobus
 de:Bushaltestelle|Haltestelle|2Bus|Autobus|ÖPNV
 hu:2Buszmegálló|2busz|megálló
 id:3Halte bus
-it:3Fermata|autobus
+it:3Fermata|5autobus|3pullman
```

Comment:
Added character limitation to "Autobus" just because I think it's better to differentiate from "auto".

Added "pullman" that it's in use in Torino.